packages/web-app-files/src/store/actions.ts

```diff
@@ -236,49 +236,21 @@ export default {
     context,
     { client, path, shareWith, shareType, permissions, role, expirationDate, storageId }
   ) {
-    if (shareType === ShareTypes.group.value) {
-      client.shares
-        .shareFileWithGroup(path, shareWith, {
-          permissions,
-          role: role.name,
-          expirationDate,
-          spaceRef: storageId
-        })
-        .then((share) => {
-          context.commit(
-            'CURRENT_FILE_OUTGOING_SHARES_UPSERT',
-            buildCollaboratorShare(
-              share.shareInfo,
-              context.getters.highlightedFile,
-              allowSharePermissions(context.rootGetters)
-            )
-          )
-          context.dispatch('updateCurrentFileShareTypes')
-          context.dispatch('loadIndicators', { client, currentFolder: path })
```

Comment:
Fyi: the issue was the `storageId` here.

Since success- and error-handling after sharing with users and groups is identical, I decided to reduce code duplication (which probably "caused" the issue in the first place).